qadence/noise/protocols.py

```diff
@@ -201,6 +197,10 @@ def dephasing(self, *args: Any, **kwargs: Any) -> NoiseHandler:
         self.append(NoiseHandler(NoiseProtocol.ANALOG.DEPHASING, *args, **kwargs))
         return self
 
-    def readout(self, *args: Any, **kwargs: Any) -> NoiseHandler:
-        self.append(NoiseHandler(NoiseProtocol.READOUT, *args, **kwargs))
+    def independent_readout(self, *args: Any, **kwargs: Any) -> NoiseHandler:
```

Comment:
Should we have a single default readout for the handler? I am not sure if this will be useful, but in case the user does not specify the type of readout, we can default it to Independent readout?

Reply:
Hm these methods are mostly helpers. So I would let them as if